instance/models/openedx_instance.py

```diff
@@ -77,6 +77,7 @@ class OpenEdXInstance(
     # Most settings/fields are inherited from mixins
 
     successfully_provisioned = models.BooleanField(default=False)
+    database_name = models.CharField(blank=True, max_length=60, default='')
```

Comment:
will be updating this to `blank=False, unique=True` in the subsequent [PR](https://github.com/open-craft/opencraft/pull/855) as mentioned here: https://github.com/open-craft/opencraft/pull/855#pullrequestreview-839088396